src/components/vmd-appointment-card.component.ts

```diff
@@ -59,15 +59,15 @@ export class VmdAppointmentCardComponent extends LitElement {
             <div class="card rounded-3 mb-5 ${classMap({clickable: this.estCliquable})}"
                  @click="${() => this.prendreRdv()}">
                 <div class="card-body">
-                    <div class="row align-items-center ">
+                    <div class="row align-items-center">
                         <div class="col">
-                            <h5 class="card-title">${Dates.isoToFRDatetime(this.lieu.prochain_rdv)}<small class="distance">${distance ? `- ${distance} km` : ''}</small></h5>
+                            <h1 class="card-title h5">${Dates.isoToFRDatetime(this.lieu.prochain_rdv)}<small class="distance">${distance ? `- ${distance} km` : ''}</small></h1>
```

Comment:
@arnaudforaison Pour des raisons de SEO, il vaut mieux éviter avoir plusieurs h1 sur une page (même si ce n'est pas interdit)

cf https://fredthewebguy.me/seo/seo-and-h1-headings-according-to-google/

Reply:
Je suis d'accord avec toi. Je me suis fait "berné" par Lighthouse. Mais je ne vois pas de h1 sur la page des rendez-vous. Au lieu de faire le changement comme ici, il faudrait revoir le rang des headings sur cette page ?

Reply:
En fait le h1 est directement dans le index.html : https://github.com/CovidTrackerFr/vitemadose-front/blob/dev/index.html#L49

Reply:
Ok mais ce n'est pas ce index.html qui est utilisé ici https://vitemadose.covidtracker.fr/centres-vaccination-covid-dpt42-loire
Ma modification intervient sur cette page (uniquement il me semble)

Reply:
Si c'est ce même `index.html`, on est sur un single page app, et c'est le index.html qui est chargée d'entrée de jeux, puis le routeur qui charge le fragment de page "variable" du site.

Reply:
Ok, mais en terme pur de rendu il n' y a pas de h1.
Il y a un h1 dans le dom mais il n'est pas rendu

Reply:
ouip, c'est une des questions que j'ai par rapport aux bots : je ne sais pas trop s'ils le trouvent car il est bien dans le DOM, le slot est bien "visible" (rien ne dit qu'il soit hidden) mais le slot n'est utilisé nulle part.